app/public/js/common/playerService.js

```diff
@@ -226,11 +226,13 @@ app.factory('playerService', function(
      * @method playPrevSong
      */
     player.playPrevSong = function() {
-
         if ( $rootScope.lock ) {
             return false;
         }
-
+        if(3 < player.elPlayer.currentTime) {
```

Comment:
You could make "3" as a constant and then use it in the code.
Like, const songTimeout = 3.
This helps other developers to reason about your code else they would have to wonder why this "3" is used.
Just a thought.